app/controllers/stories_controller.rb

```diff
@@ -345,9 +345,7 @@ def set_user_json_ld
       image: Images::Profile.call(@user.profile_image_url, length: 320),
       name: @user.name,
       email: @user.setting.display_email_on_profile ? @user.email : nil,
-      jobTitle: @user.employment_title.presence,
```

Comment:
This change was slated for https://github.com/forem/forem/pull/14079 but I decided to break that down into smaller pieces, starting with removing some fields.